PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/runner/webappconfig/WebAppRunState.java

```diff
@@ -66,11 +68,31 @@ public WebAppBase executeSteps(@NotNull RunProcessHandler processHandler
         }
         webAppConfiguration.setTargetName(file.getName());
         WebAppBase deployTarget = getDeployTargetByConfiguration(processHandler);
+        // update settings
+        if (MapUtils.isNotEmpty(webAppConfiguration.getApplicationSettings())) {
+            updateApplicationSettings(deployTarget, processHandler);
+        }
         WebAppUtils.deployArtifactsToAppService(deployTarget, file,
                 webAppConfiguration.isDeployToRoot(), processHandler);
         return deployTarget;
     }
 
+    private void updateApplicationSettings(WebAppBase deployTarget, RunProcessHandler processHandler) {
+        if (deployTarget instanceof WebApp) {
+            processHandler.setText("Updating Application Settings...");
+            WebApp webApp = (WebApp) deployTarget;
+            AzureWebAppMvpModel.getInstance().updateWebAppSettings(webAppSettingModel.getSubscriptionId(),
+                    webApp.id(), webAppConfiguration.getApplicationSettings(), new HashSet<>());
+            processHandler.setText("Updated Application Settings successfully.");
+        } else if (deployTarget instanceof DeploymentSlot) {
+            processHandler.setText("Updating Application Settings...");
```

Comment:
Move this statement ahead in case duplicated

Reply:
it should not out of if/else if block. wen can not move.